web/lib/service/analytics.ts

```diff
@@ -12,6 +12,29 @@ init(ENV_CONFIG.amplitudeApiKey ?? '', undefined, { includeReferrer: true })
 
 export { track }
 
+// Integrate Sprig
+
+try {
+  (function(l,e,a,p) {
+    // @ts-expect-error
```

Comment:
<img width="1209" alt="image" src="https://user-images.githubusercontent.com/856034/192307926-05f750da-cfc5-4a6f-818c-6dd88882d3cf.png">

Currently getting these errors. Could fix, but I think installing the [Sprig NPM package](https://docs.sprig.com/docs/web-javascript#install-with-npmyarn-beta) might just be better